app/scripts/views/settings/communication_preferences.js

```diff
@@ -93,12 +94,23 @@ define(function (require, exports, module) {
     },
 
     setOptInStatus (newsletterId, isOptedIn) {
-      var method = isOptedIn ? 'optIn' : 'optOut';
+      let method, eventPrefix;
+      if (isOptedIn) {
+        method = 'optIn';
+        eventPrefix = '';
+      } else {
+        method = 'optOut';
+        eventPrefix = 'un';
+      }
+
       this.logViewEvent(method);
 
       return this.getMarketingEmailPrefs()[method](newsletterId)
         .then(() => {
           this.logViewEvent(method + '.success');
+          // Emit an additional flow event for consistency with
+          // the call to optIn in the account model
+          this.logFlowEvent(`newsletter.${eventPrefix}subscribed`);
```

Comment:
Let's think about how we can consolidate metrics, ubiquitous language should apply to metric event names too! We have multiple places where both View events and Flow events are logged:

<img width="456" alt="screen shot 2017-06-20 at 10 55 53" src="https://user-images.githubusercontent.com/848085/27327742-70321df0-55a7-11e7-9da3-a3681e183f84.png">

`settings.communication-preferences.optIn.success` and `flow.newsletter.subscribed` mean the same thing yet have two distinct names, this is a recipe for confusion.